doc/content/gateways/moko-mkgw2-lw/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"
+description: ""
+---
+
+MOKOSMART's 8 channel MKGW2 gateway is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for indoor use and is easy to install.
+
+{{< figure src="mkgw2-lw.jpg" alt="mkgw2-lw" class="plain float" >}}
+
+## Requirements
+
+1. User account on {{% tts %}} with rights to create gateways.
+2. MOKOSMART MKGW2-LW connected to the internet via Ethernet or WiFi. (For Ethernet and WiFi settings, see sections 5 and 6 of the [user manual](https://www.mokosmart.com/lorawan-gateway-mkgw2-lw/))
+
+## Registration
+
+Create a gateway by following [the instructions]({{< ref "/gateways/adding-gateways" >}}). 
+
+The **EUI** of the gateway can be found on the configuration web page of the gateway. 
+
+{{< figure src="eui.png" alt="Gateway EUI" >}}
+
+## Configuration via Browser
```

Comment:
This is not really a configuration, but accessing the gateway via web platform only. I think this section should come before Registration, because finding the EUI in the web management platform requires to be able to access it first

Reply:
We always add the gateway in the console first. For Basic Station, you need to generate an API key for the device to configure it. The EUI is also printed on the packaging, @Fakerallen just didn't mention it.